test/quadratic.jl

```diff
@@ -0,0 +1,14 @@
+using IntervalArithmetic, IntervalRootFinding
+
+@testset "Quadratic Interval Equations" begin
+    a = [interval(1, 5), interval(-1, 1), interval(1, 2), ]
+    b = [interval(2, 4), interval(-2, 2), interval(1, 3), ]
+    c = [interval(0, 2), interval(-1, 1), interval(2, 6), ]
+    x = [[interval(-4, -2), interval(-0, -0)], [interval(-∞, -1), interval(-1, -1), interval(-1, ∞)], [interval(-2, -1)], ]
+    l = [2, 3, 1, ]
+    for i in 1:length(a)
+            roots = quadratic_interval(a[i], b[i], c[i])
```

Comment:
I think it would be clearer to have each test case as a separate object, e.g. a list of objects of the form

`( (a, b, c), [roots] )`

or even make an actual `struct` for each test case.

Note that you don't need a separate `l` if you have the root vectors.